public/js/actions/subscriptions.js

```diff
@@ -98,11 +100,20 @@ export function _createSubscription({dispatch, productId,
     console.log('Successfully subscribed + completed payment');
     dispatch(transactionActions.complete({userEmail: email}));
   }).fail($xhr => {
-    if (data.pay_method_nonce) {
-      dispatch({
-        type: actionTypes.CREDIT_CARD_SUBMISSION_ERRORS,
-        apiErrorResult: $xhr.responseJSON,
-      });
+    console.log($xhr);
+    if ($xhr.status === 400 && $xhr.responseJSON &&
+        $xhr.responseJSON.error_response) {
```

Comment:
I bet we'll need this again for another action so we should make a mental note to wrap this up somehow (later).